src/main/scala/com/github/simplesteph/ksm/source/BitbucketServerSourceAcl.scala

```diff
@@ -80,8 +88,11 @@ class BitbucketServerSourceAcl extends SourceAcl {
             s"$protocol://$hostname:$port/projects/$project/repos/$repo/browse/$filePath?raw"
 
           val fileRetrievalRequest = new Request(rawRetrieveUrl)
+          if (branch.isDefined) {
```

Comment:
I think the scala way is something like `branch.forEach { b =>  fileRetrievalRequest.queryParam("at", b) }` but please check too

Reply:
looks better, thanks. Done